libvast/include/vast/address.hpp

```diff
@@ -86,6 +88,14 @@ class address : detail::totally_ordered<address>, detail::bitwise<address> {
     return result;
   }
 
+  /// Construct a pseudonymized address using the Crypto-PAn algorithm.
+  /// @param original The address to be pseudonymized.
+  /// @param seed 256-bit seed for the cipher and padding.
+  /// @returns A copy of the `original` address with pseudonymized bytes.
+  static address pseudonymized(
+    const address& original,
+    const std::array<byte_type, pseudonymization_seed_array_size>& seed);
```

Comment:
Most of the functions in our code base are verbs, while classes are usually nouns. In this case, I think it makes sense to name the function `pseudonymize`, and to make it a friend of `address` instead of a static member function.

Reply:
I oriented myself according to the static `v4()` and `v6()` methods (is `v6` a verb?). I renamed the method to `pseudonymize` in

Reply:
The friend just reduces the function overload set, this is a small win for compilation time.